### PR TITLE
Improve text in examples, avoid hf tokenizer warnings and more links

### PR DESCRIPTION
**Pull request by jobergum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jobergum** [comment]
This increases the pipeline latency as it encodes more documents, but the examples are now more inspiring.

File: docs/sphinx/source/examples/colpali-document-retrieval-vision-language-models-cloud.ipynb [hunk 1]
```diff
@@ -14,6 +14,8 @@
                 "\n",
                 "# Vespa 🤝 ColPali: Efficient Document Retrieval with Vision Language Models\n",
                 "\n",
+                "For a simpler example of using ColPali, where we use one Vespa document = One PDF page, see [simplified-retrieval-with-colpali-vlm_Vespa-cloud](simplified-retrieval-with-colpali-vlm_Vespa-cloud.ipynb).\n",
```
Comment (thomasht86) [inline docs/sphinx/source/examples/colpali-document-retrieval-vision-language-models-cloud.ipynb:17]:
```suggestion
                "For a simpler example of using ColPali, where we use one Vespa document = One PDF page, see [simplified-retrieval-with-colpali-vlm_Vespa-cloud](https://pyvespa.readthedocs.io/en/latest/examples/simplified-retrieval-with-colpali-vlm_Vespa-cloud.html).\n",
```

File: docs/sphinx/source/examples/colpali-document-retrieval-vision-language-models-cloud.ipynb [hunk 2]
```diff
@@ -41,9 +43,12 @@
...
                 "where we represent the colbert embeddings per document with the tensor `tensor(page{}, patch{}, v[128])`. This enables \n",
-                "us to use the PDF as the document (retrievable unit), storing the page embeddings in the same document.\n",
+                "us to use the PDF as the document (retrievable unit), storing the page embeddings in the same document. \n",
+                "\n",
+                "The upside of this is that we do not need to duplicate document level meta data like title, url, etc. But, the downside is that \n",
+                "we cannot retrieve using the ColPali embeddings directly, but need to use the extracted text for retrieval. The ColPali embeddings are only used for reranking the results. \n",
                 "\n",
-                "For a simpler example where we use one vespa document = One PDF page, see [this notebook](simplified-retrieval-with-colpali-vlm_Vespa-cloud.ipynb).\n",
+                "For a simpler example where we use one vespa document = One PDF page, see [simplified-retrieval-with-colpali-vlm_Vespa-cloud](simplified-retrieval-with-colpali-vlm_Vespa-cloud.ipynb).\n",
```
Comment (thomasht86) [inline docs/sphinx/source/examples/colpali-document-retrieval-vision-language-models-cloud.ipynb:51]:
```suggestion
                "For a simpler example where we use one vespa document = One PDF page, see [simplified-retrieval-with-colpali-vlm_Vespa-cloud](https://pyvespa.readthedocs.io/en/latest/examples/simplified-retrieval-with-colpali-vlm_Vespa-cloud.html).\n",
```

**Review by thomasht86 — APPROVED** [review]
This is great. 💯 
Highlights the benefits of Visual retrieval much better!

File: docs/sphinx/source/examples/colpali-document-retrieval-vision-language-models-cloud.ipynb (hunk 1 above)
Comment (jobergum) [reply docs/sphinx/source/examples/colpali-document-retrieval-vision-language-models-cloud.ipynb:17]: Doesn't sphinx auto solve this?

Comment (thomasht86) [reply docs/sphinx/source/examples/colpali-document-retrieval-vision-language-models-cloud.ipynb:17]: I think it does if you link the html, but with the `.ipynb`-suffix, I think you get the raw notebook - eg. [https://pyvespa.readthedocs.io/en/latest/examples/simplified-retrieval-with-colpali-vlm_Vespa-cloud.ipynb](https://pyvespa.readthedocs.io/en/latest/examples/simplified-retrieval-with-colpali-vlm_Vespa-cloud.ipynb)

Comment (jobergum) [reply docs/sphinx/source/examples/colpali-document-retrieval-vision-language-models-cloud.ipynb:17]: Thanks

**Comment by jobergum** [comment]
Let us get this merged

**Review by thomasht86 — APPROVED** [review]
Agreed. I will follow up on failing notebooks - some openai-related errors.

**Pull request merged**